configuration/configuring-root-certificates/en.md

```diff
@@ -0,0 +1,119 @@
+
+### List Database Entities
+
+`$ certutil -d sql:$HOME/.pki/nssdb -L`
+
+#### Example Output
+
+This example shows an existing self-signed, trusted CA certificate as described by the SSL Trust Attribute `C`.
+
+```
+Certificate Nickname                                         Trust Attributes
+                                                             SSL,S/MIME,JAR/XPI
+
+mynew-Root-CA                                                C,,
+```
+
+### Add a Certificate
+
+`$ certutil -d sql:$HOME/.pki/nssdb -A -t "P,," -n <cert-name> -i <path_to_crt_file>`
+
+#### Example Output
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -A -t "C,," -n mynew-Root-CA -i mynew-Root-CA.CRT
+$ certuilt -d sql:$HOME/.pki/nssdb -L
+Certificate Nickname                                         Trust Attributes
+                                                             SSL,S/MIME,JAR/XPI
+
+mynew-Root-CA                                                C,,
+```
+
+### Validate the Certificate
+
+`$ certutil -d sql:$HOME/.pki/nssdb -V -u <certusage> -n <cert_name>`
+
+#### Example Output
+
+In this example, we specify the usage context with the `-u` option to "Verify CA" `Y`.
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -V -n mynew-Root-CA -u Y
+certutil: certificate is valid
+```
+
+In this example, we try to validate the Root/CA certificate in the SSL client usage context.
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -V -n mynew-Root-CA -u C
+certutil: certificate is invalid: Certificate key usage inadequate for attempted operation.
+```
+
+### Using a Certificate
+
+Some client applications will need to be restarted to use the new CA certificate. Valid SSL or TLS host/service certificates including a signature with by the newly installed CA/Root certificate can now be validated and inspected. Web browsers provide utilities for certificate inspection, often with a status icon in the URL bar. 
+
+A command line method of inspecting a certificate: `curl https://protected.resource.uri -vI --stderr -`
+
+### Deleting a Certificate in the Database
+
+`$ certutil -d sql:$HOME/.pki/nssdb -D -n <cert-name>`
+
+#### Example Output
+
+```
+$ certuilt -d sql:$HOME/.pki/nssdb -L
+Certificate Nickname                                         Trust Attributes
+                                                             SSL,S/MIME,JAR/XPI
+```
+
+## Advanced Topics
+
+### Certificate Types
+
+SSL/TLS and SSH certificates are separate and unique concepts with distinct use-cases. This article did not provide details on hosting or serving network resources with a non-public certificate, as most implementations are service dependent (eg: Apache vs MongoDB) and are generally independent from the Solus' Public/Private Key Infrastructure (PKI).
```

Comment:
```suggestion
SSL/TLS and SSH certificates are separate and unique concepts with distinct use-cases. This article did not provide details on hosting or serving network resources with a non-public certificate, as most implementations are service dependent (eg: Apache vs MongoDB) and are generally independent from Solus' Public/Private Key Infrastructure (PKI).
```

Lose a word here. I guess "the Solus" (without apostrophe) would also be possible.